File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dic = {}
        for n in nums:
            dic[n] = 1 + dic.get(n, 0)
        return list(sorted(dic.keys(), key=lambda x: dic[x], reverse=True))[:k]

File: Data Structures & Algorithms/duplicate-integer/submission-2.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        HashSet<int> s = new HashSet<int>();
        for (int x = 0; x < nums.Length; x++){
            if (s.Contains(nums[x])){
                return true;
            }else{
                s.Add(nums[x]);
            }
        }
        return false;

    }
}